pull-request.py

```diff
@@ -102,9 +97,14 @@ def create_pull_request(
                 print("Pull request from %s to %s is already open!" % (source, target))
                 is_open = True
 
+                # Does the user want to pass if the pull request exists?
```

Comment:
Yikes, I was definitely tired when I opened the previous PR to completely put this in the wrong spot. It's rather embarrassing really ;)

Reply:
Lol no worries